packages/core/test/specs/data_sources/__snapshots__/jsonplaceholder.ts.snap

```diff
@@ -0,0 +1,76 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`JsonPlaceholder Usage should render a list of comments from jsonplaceholder api 1`] = `
+"<body>
+  <div>
+    <h4>
+      <div>id labore ex et quam laborum</div>
+    </h4>
+    <p>
+    <div>1</div>
```

Comment:
@artf I wonder if we should allow people to change the dom component that gets injected, see here its always a `div` maybe we could allow in the data variable a way to specify the component to be injected like: 


```ts
editor.addComponents([
  {
    tagName: 'h1',
    type: 'text',
    components: [
      {
        type: 'data-variable',
        defaultValue: 'default',
        path: 'my-datasource.id1.content',
        domComponent: "div" <----- default
      },
    ],
  },
]);
```

Or maybe you don't want to inject a div inside a header so you change to a span. 


```ts
editor.addComponents([
  {
    tagName: 'h1',
    type: 'text',
    components: [
      {
        domComponent: "span"
      },
    ],
  },
]);
```

Reply:
this should already work:
```js
{ 
type: 'data-variable', 
defaultValue: 'default', 
path: 'my-datasource.id1.content', 
tagName: "span"
},
```